scripts/split.sh

```diff
@@ -46,19 +46,27 @@ do
   shift
 done
 
+if [ -z "$LC0LOCKFILE" ]
+then
+  echo "env var LC0LOCKFILE not set"
+  exit 1
+fi
 
 # clear test and train split dirs
-rm -rf $TESTDIR $TRAINDIR
-mkdir -vp $TESTDIR $TRAINDIR
+if [ ! -d "$TESTDIR" ] || [ ! -d "$TRAINDIR" ]
+then
+  rm -rf $TESTDIR $TRAINDIR
+  mkdir -vp $TESTDIR $TRAINDIR
+fi
 
-n=0
+let n="$(ls $TESTDIR | wc -l) + $(ls $TRAINDIR | wc -l)"
 let overhead="$WINSIZE / 10"
 let max="$WINSIZE + $overhead"
```

Comment:
I was thinking about this some more today - I think you probably should make this slightly larger than winsize + overhead - to have some slop in case the split isn't mathematically exact and the training script complains about not having enough chunks as a result.

Reply:
Yeah I agree, it should never fall under the window size. So + some constant e.g. 5?

Reply:
I think 100 should be an obviously comfortable margin given the way the split is calculated - if we were using sha it would nee to be larger and probably would depend on the window size.

Reply:
So winsize + overhead + 100?

Reply:
I think that should be good enough yes.